template/module/README.rst

```diff
@@ -77,16 +70,24 @@ The development of this module has been financially supported by:
 * Company 2 name
 
 Maintainer
-----------
+===========
+
+.. ATTENTION: replace the {placeholders} below
+.. project_repo is the GithHun project name. For example "maintainer-tools"
```

Comment:
Do you have anything against [Huns](https://duckduckgo.com/?q=hun&t=ffnt&iax=1&ia=images)?